extensions/theme-seti/icons/vs-seti-icon-theme.json

```diff
@@ -1240,6 +1240,13 @@
 		"_zip_1": {
 			"fontCharacter": "\\E091",
 			"fontColor": "#6d8086"
+		},
+		// {{SQL CARBON EDIT}}
```

Comment:
I don't think comments are strictly valid in JSON files.  Though a lot of parsers will ignore them, so if this works then it's fine for me.

Reply:
It's an interesting one since the other built-in theme has this. It's not breaking it (likely because we use a good parser) so for now I think this is a good help?